src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -221,29 +167,15 @@ public void RequestAnalysis(string path, string charset, IEnumerable<SonarLangua
             }
         }
 
-        public void Accept(string path, IEnumerable<Issue> issues)
```

Comment:
_Accept_ is the method on _IIssueConsumer_, the callback interface used by the daemon to return results. I've made the _TextBufferIssueHandler_ implement the interface directly (previously this class used to receive the results then then forward them to the issue handler).